util/teststorage/appender.go

```diff
@@ -65,13 +65,17 @@ func (s Sample) String() string {
 	// Print all value types on purpose, to catch bugs for appending multiple sample types at once.
 	h := ""
 	if s.H != nil {
-		h = s.H.String()
+		h = "" + s.H.String()
```

Comment:
why this empty string addition?

Reply:
Ups, should be space! Thanks for noticing.

Help with readability, I can use fmt.Sprintf too